### PR TITLE
Feature/python3 compatibility

### PR DESCRIPTION
**Pull request by benjamts**
Version 0.2.1 of plaid-python is not python3 compatible. Use the 2to3 script to update the package. Also fix some tests that were either broken or provided unreliable coverage and address some outstanding PEP8 issues.

### CODE REVIEW

**Comment by linuxlewis** [comment]
I might leave out the last commit but besides that :+1:

File: plaid/__init__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
-__version__ = '0.2.1'
+__version__ = '1.0.0'
```
Comment (rgreinho) [inline plaid/__init__.py]: That's a big version bump...

File: plaid/__init__.py [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
 
-from client import Client, require_access_token
+from .client import Client, require_access_token  # NOQA
```
Comment (rgreinho) [inline plaid/__init__.py]: Why not putting one import per line like recommended by OpenStack (http://docs.openstack.org/developer/hacking/#imports) and get rid of #NOQA?

Also, you should not need the ``.`` before ``Client``.

File: plaid/client.py [hunk 3]
```diff
@@ -1,11 +1,12 @@
 import json
-from urlparse import urljoin
+from urllib.parse import urljoin
 
-from http import http_request
+from .http import http_request
```
Comment (rgreinho) [inline plaid/client.py]: ``plaid.http``. seems cleaner to me.

File: samples/gae/main.py [hunk 4]
```diff
@@ -19,8 +20,8 @@ def get(self):
         w("<body>")
         w("<pre>")
         client = plaid.Client(CLIENT, SECRET, ACCESS_TOKEN)
-        #res = client.transactions()
-        #print res.content
+        # res = client.transactions()
+        # print res.content
```
Comment (rgreinho) [inline samples/gae/main.py]: Remove dead code.

**Comment by rgreinho** [comment]
Great job overall :+1:

**Comment by rgreinho** [comment]
Regarding the version number I would open the discussion with the maintainers to see what they think.

**Comment by rgreinho** [comment]
Regarding the packaging I would repackage the application to use PBR and use the wheel format.

File: plaid/__init__.py (hunk 2 above)
Comment (benjamts) [inline plaid/__init__.py]: I prefer multiple `from subprocess import foo, bar` on one line, which is okay by PEP8

https://www.python.org/dev/peps/pep-0008/#imports

**Comment by benjamts** [comment]
I'm going to clean these changes up, split them into multiple PRs, and make requests into the original repository's master branch.

**Pull request closed without merging**